src/proctored-exam-settings/proctoredExamSettings.scss

```diff
@@ -3,3 +3,10 @@ legend {
     font-size: 1.0rem;
     font-weight: 700;
 }
+
+.form-check:focus-within {
```

Comment:
Is it intended for the label to also be styled this way? It looks like the other form elements on this page (e.g. the proctoring provider field, the email field) only visually style the actual input element. Was it possible to replicate this kind of styling for the checkboxes and radio buttons?

Reply:
I had intended it to be styled that way, mostly because if I add a focus indicator to the checkbox or radio button alone, I have to use `outline` instead of `border`. `outline` doesn't have a radius property associated with it, so it wouldn't be able to have rounded edges like the other elements.

Reply:
If you are curious about what the outline would look like, you can add this: 

`input[type="radio"]:focus,
input[type="checkbox"]:focus {
  outline: 1px solid #6582dd;
}`